assets/src/edit-story/output/utils/styles.js

```diff
@@ -102,6 +102,12 @@ function CustomStyles() {
                 bottom: 0;
                 margin: 0;
               }
+
+              @media (prefers-reduced-motion: no-preference) {
+                .animation-wrapper {
+                  transform: var(--ws-initial-transform);
+                }
+              }
```

Comment:
I'm actually not seeing any initial frame issues if we remove setting the initial frame entirely. Maybe this was only needed with how the old amp previewer rendered a story and with the latest update is no longer needed.

Can you confirm on your local that if you remove this and preview a story with a bunch of animations that there's no flicker on initial frame issues where the element is set to the last keyframe style for the first frame, then plays the animation? @choumx @swissspidy